gbfs-validator/README.md

```diff
@@ -0,0 +1,38 @@
+# GBFS-Validator CLI
+
+[![npm version](https://badge.fury.io/js/gbfs-validator.svg)](http://badge.fury.io/js/gbfs-validator)
+
+GBFS Validator is a command-line tool for validating General Bikeshare Feed Specification (GBFS) data feed and outputing the report in a JSON file
+
+## Add the Dependency
+
+To use the `gbfs-validator` cli in your own project, you need to
+first install our [Node.js npm package](https://www.npmjs.com/package/gbfs-validator):
+
+```
+npm install gbfs-validator
+```
+
+## Usage
```

Comment:
[suggestion]: Add a section on how to use the libraries within a javascript project as a code